Framework/PythonInterface/plugins/algorithms/InelasticEMUauReduction.py

```
@@ -149,8 +149,8 @@ def PyInit(self):
 
     def PyExec(self):
         # add the following to where it should break
-        # debugpy.wait_for_client()
-        # debugpy.breakpoint()
+        debugpy.wait_for_client()
```

Ruff (0.12.2)

Trace found: `debugpy.wait_for_client` used

(T100)